src/domain/usecases/CopyDataSetsOUUserCase.ts

```diff
@@ -0,0 +1,96 @@
+import _ from "lodash";
+import { DataSetsRepository } from "domain/repositories/DataSetsRepository";
+import { PostOptions, Ref } from "types/d2-api";
+import { DataSet } from "domain/entities/DataSet";
+import { OUCopyResult } from "domain/entities/OUCopyResult";
+
+export class CopyDataSetsOUUserCase {
+    constructor(
+        private dataSetsRepository: DataSetsRepository
+    ) { }
```

Comment:
I see `prettier` is installed, but there is no `prettify` command, added. Now `yarn prettify` applies the code format. 

It's usually very handy to configure vscode to automatically apply format on save. Setting: "Format on Save"

Reply:
I have Format on Save activated, just needed to switch to Prettier formatter.